src/main/scala/nodes/learning/KernelModel.scala

```diff
@@ -0,0 +1,78 @@
+package nodes.learning
+
+import breeze.linalg._
+import edu.berkeley.cs.amplab.mlmatrix.{RowPartition, NormalEquations, BlockCoordinateDescent, RowPartitionedMatrix}
+import nodes.stats.{StandardScalerModel, StandardScaler}
+import org.apache.spark.rdd.RDD
+import org.apache.spark.broadcast.Broadcast
+import nodes.util.{VectorSplitter, Identity}
+import utils.{MatrixUtils, Stats, KernelUtils}
+import workflow.{Transformer, LabelEstimator}
+
+
+/**
+ * @param models different kernel models for different lambdas
+ * @param blockSize blockSize to split data before applying transformations
+ * @param lambdas  lambdas corresponding to the models
+ * @param trainData the training Data
+ * @param kernelGen the kernel generator
+ **/
+
+class KernelBlockModel(
+  models: Broadcast[Array[DenseMatrix[Double]]],
+  blockSize: Int,
+  lambdas: Array[Double],
```

Comment:
This multi-lambda in a Transformer thing is a little unconventional, because it means to get a single prediction out someone needs to chain on another transformer. It's probably fine but we should decide how we want these interfaces to look.